lib/core-integration/src/Test/Integration/Scenario/API/Byron/Migrations.hs

```diff
@@ -207,7 +208,7 @@ spec = do
               testAddressCycling ctx 3
               testAddressCycling ctx 10
 
-    it "BYRON_MIGRATE_01 - \
+    Hspec.it "BYRON_MIGRATE_01 - \
```

Comment:
we want to revert this to our `it`, right?

Reply:
Using regular `Hspec.it` makes sense here I think, because those test use specific mnemonic wallets, so in case when those test fail the subsequent run will result in `409` - wallet already exists anyway...

Reply:
> Using regular Hspec.it makes sense here I think, because those tests use specific mnemonic wallets

Indeed. We don't want to retry these tests because they use a hard-coded mnemonic. So I changed it back to the standard hspec "it" to make debugging easier.

Reply:
What about prefixing the retrying version `it` with `sh` to clarify its function? (sorry, couldn't resist :grin:)